course3/homework-string/my_string.cc

```diff
@@ -0,0 +1,64 @@
+#include <iostream>
+#include <cstring>
+#include "my_string.h"
+
+MyString::MyString(){
+	buffer = std::unique_ptr<char[]>(new char[1]);
+	buffer = '\0';
```

Comment:
I also tried:
buffer = std::unique_ptr<char[]>(new char('\0'));

But valgrind complained about: Mismatched free() / delete / delete [] 
and even creating a destructor didn't make that warning go away.

As written, the only warning in my compiler is regarding for loops:
  my_string.cc:14:16: warning: comparison between signed and unsigned integer expressions [-Wsign-compare]   for (i = 0; i < std::strlen(theString); i++){